### DISCUSSION TITLE
[Questions] Inpaint GUI hotkey in 2.6.5 and other stuff ?

### DISCUSSION THREAD

**Discussion by vecuccio** [mashb1t/Fooocus#80] [Q&A] [answered]
Hi, Until literally afew minutes ago, I thought that v2.5.5 was the latest Foocus, only upon seeing a random random Reddit post did I even find out there was a v2.6.5 made by mashb1t, and then I found out here that he has retired too :-) 

I posted a similar question on the GitHub of the original creator of Fooocus afew days ago, but then read he has been silent for a long time (so it remains unanswered, as probably will this be)

Please bear with me, I am a complete n00b to AI, and am trying to learn stuff as fast as I can, so if my questions appear n00by because I use incorrect terms, that's why :-)

I was wondering, is it worth upgrading ? The only issues I have with v2.5.5 are- 

-That there is no hotkey to turn off the in-paining GUI - which gets in the way if you are trying to paint the top right hand corner.
- And that there is no 'Undo Paint' brush to 'unpaint' an area you have incorrectly painted.
- Has the 'Enhance' feature workflow been streamlined ? It is a nightmare of nested menus. And the feature doesn't seem to enhance an entire photo like Krea.AI, just various body parts one by one, and then it changes the appearance of a face while doing it - which apparently, maybe, can be remedied with an intimate knowledge of certain sliders in the debug menu, which is way above my current pay grade.
- About all the sliders. Are there any 'i' icons you can hover over that tell you what each slider does ? Or improved documentation n general ?

Have any of these things been improved/fixed/added in 2.6.5 ?

Thanks for reading my post. :-)

**Comment by mashb1t** [comment, marked as answer]
2.5.5 is the latest official version of Fooocus, and the latest version of this repository is 2.6.5. I've synced almost all changes to upstream, but left the prompt ttanslation and some other minor things in this repository only.

> Have any of these things been improved/fixed/added in 2.6.5 ?

No.

> -That there is no hotkey to turn off the in-paining GUI - which gets in the way if you are trying to paint the top right hand corner.

This is correct, there is no hotkey

> And that there is no 'Undo Paint' brush to 'unpaint' an area you have incorrectly painted.

This has to do with using Gradio 3.42.1, we're stuck with this and upgrading takes too much time, which none of the contributors has.

> Has the 'Enhance' feature workflow been streamlined ? It is a nightmare of nested menus. And the feature doesn't seem to enhance an entire photo like Krea.AI, just various body parts one by one, and then it changes the appearance of a face while doing it - which apparently, maybe, can be remedied with an intimate knowledge of certain sliders in the debug menu, which is way above my current pay grade.

It is well streamlined as it is. It's not the purpose of the enhance feature to enhance the whole image, please use vary subtle or strong in tab upscale or  vary for this for this.

> About all the sliders. Are there any 'i' icons you can hover over that tell you what each slider does ? Or improved documentation n general ?

As you can see, there are no info texts. In-depth documentation can be found by following the documentation links in the UI or by looking into official discussions in upstream where explanations can be found. I can also recomment Kleebz Tech youtube channel who does/did create in-depth feature showcases.

**Reply by vecuccio** [reply]
Thanks for taking the time to reply mashb1t
